mitodl/open-discussions
# Added page for users who aren't logged in

#### What are the relevant tickets?
Fixes #200 

#### What's this PR do?
Redirects users without a JWT to `/auth_required/` to see a message about logging in. There is a link which will redirect the user to micromasters.

#### How should this be manually tested?
There is a new environment variable `OPEN_DISCUSSIONS_EXTERNAL_LOGIN_URL` which is a link to micromasters.

Open the browser in incognito mode and verify that you are redirected to `/auth_required/` no matter what path you choose. If you click on the link you should be directed to micromasters.
